lib/main.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+require('string.prototype.startswith');
```

Comment:
This will add it to the prototype of `String` across the runtime? I would suggest to use a static method and avoid touching a primitive's prototype.